void tomo_db0::IoLoop::run_loop<false, true, false, false, false>()
--- PRE

+++ POST

@@ -609,7 +609,7 @@

 0b4d [ 3] nopl (%rax)
 0b50 [ 5] mov $0x1,%esi
 0b55 [ 3] mov %rbx,%rdi
-0b58 [ 5] call <tomo_db0::IoLoop::arm_accept(tomo_db0::UrKind)>
+0b58 [ 5] call <tomo_db0::IoLoop::fliplisten_retry_accept(tomo_db0::UrKind)>
 0b5d [ 5] jmp <+0x2fb>
 0b62 [11] data16 cs nopw 0x0(%rax,%rax,1)
 0b6d [ 3] nopl (%rax)
